cryptoki/src/error/mod.rs

```diff
@@ -131,11 +133,11 @@ impl From<std::convert::Infallible> for Error {
     }
 }
 
-impl From<RvError> for Error {
-    fn from(rv_error: RvError) -> Self {
-        Error::Pkcs11(rv_error)
-    }
-}
+// impl From<RvError> for Error {
```

Comment:
If it's not used I guess it can be removed.

Reply:
Yeah, sorry, commented out when I was fixing things, forgot to delete it 😁